src/clients/alternatives.rb

```diff
@@ -1,3 +1,12 @@
 require "y2_alternatives/dialog/list_alternatives"
 
-Y2Alternatives::Dialog::ListAlternatives.run
+Yast.import "Confirm"
+
+# Checks if user is root and create a ListAlternatives dialog
+class AlternativesClient
```

Comment:
I would prefer to have this in a class `Y2Alternatives::Client::Alternatives` (or any other name), with this file containing only the call.